lib/matplotlib/dates.py

```diff
@@ -83,10 +83,11 @@
      Out[1]: 732401
 
 All the Matplotlib date converters, tickers and formatters are timezone aware.
-If no explicit timezone is provided, :rc:`timezone` is assumed.  If you want to
-use a custom time zone, pass a `datetime.tzinfo` instance with the tz keyword
-argument to `num2date`, `.Axis.axis_date`, and any custom date tickers or
-locators you create.
+If no explicit timezone is provided, :rc:`timezone` is assumed, provided as a
+string.  If you want to use a custom timezone, pass a `datetime.tzinfo`
+instance or a string with the timezone name to the tz keyword argument of
+`num2date`, `.Axis.axis_date`, and any custom date tickers or locators you
+create. Timezone strings are parsed by `~dateutil.tz.gettz`.
```

Comment:
```suggestion
string.  If you want to use a different timezone, pass the *tz* keyword argument 
of `num2date` to any date tickers or locators you create.  This can be either a 
`datetime.tzinfo` instance or a string with the timezone name that can be parsed
by `~dateutil.tz.gettz`.
```

(an awkward sentence can usually be avoided with a re-org ;-) )

Reply:
Thanks! Added that (and finally got the plain dateutil-link to work in a reasonable way).